M2/BUILD/docker/rhel/Dockerfile

```diff
@@ -18,9 +18,9 @@ RUN dnf install -y 'dnf-command(config-manager)' && \
 RUN dnf -y install autoconf automake bison boost-devel bzip2 cmake     \
     diffutils eigen3 flex gc-devel gcc-c++ gcc-gfortran gdbm-devel git \
     glpk-devel gmp-devel gtest-devel lapack-devel libffi-devel libtool \
-    libxml2-devel make mpfr-devel ncurses-devel openblas-devel patch   \
-    python3-devel R readline-devel rpm-build tbb-devel which xz-devel  \
-    zlib-devel
+    libxml2-devel make mpfr-devel ncurses-devel ninja-build            \
```

Comment:
What's ninja used for?

Reply:
Oh I see you just want this docker to work for both cmake and autotools build, never mind.